### PR TITLE
fix warnings

### PR DESCRIPTION
**Pull request by nick2525**
## Summary
Just fix some warnings

## Link to pull request in Documentation repository

Documentation: home-assistant/companion.home-assistant#

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @TESTPERSONAL, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;android#1867).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: app/src/main/java/io/homeassistant/companion/android/sensors/LightSensorManager.kt [hunk 1]
```diff
@@ -82,16 +82,14 @@ class LightSensorManager : SensorManager, SensorEventListener {
...
-            if (event.sensor.type == Sensor.TYPE_LIGHT) {
-                onSensorUpdated(
-                    latestContext,
-                    lightSensor,
-                    event.values[0].roundToInt().toString(),
-                    "mdi:brightness-5",
-                    mapOf()
-                )
-            }
+        if (event != null && event.sensor.type == Sensor.TYPE_LIGHT) {
```
Comment (JBassett) [inline app/src/main/java/io/homeassistant/companion/android/sensors/LightSensorManager.kt:85]: Think this is even better?
```suggestion
        if (event?.sensor.type == Sensor.TYPE_LIGHT) {

```

File: app/src/main/java/io/homeassistant/companion/android/sensors/ProximitySensorManager.kt [hunk 2]
```diff
@@ -80,24 +80,21 @@ class ProximitySensorManager : SensorManager, SensorEventListener {
...
-                        sensorValue
-                onSensorUpdated(
-                    latestContext,
-                    proximitySensor,
-                    state,
-                    "mdi:leak",
-                    mapOf()
-                )
-            }
+        if (event != null && event.sensor.type == Sensor.TYPE_PROXIMITY) {
```
Comment (JBassett) [inline app/src/main/java/io/homeassistant/companion/android/sensors/ProximitySensorManager.kt:83]:
```suggestion
        if (event?.sensor.type == Sensor.TYPE_PROXIMITY) {

```

File: app/src/main/java/io/homeassistant/companion/android/sensors/SensorWorker.kt [hunk 3]
```diff
@@ -73,9 +73,8 @@ class SensorWorker(
             val foregroundInfo = ForegroundInfo(NOTIFICATION_ID, notification)
             setForeground(foregroundInfo)
             val lastUpdateSensor = sensorDao.get(LastUpdateManager.lastUpdate.id)
-            if (lastUpdateSensor != null) {
-                if (lastUpdateSensor.enabled)
-                    LastUpdateManager().sendLastUpdate(appContext, TAG)
+            if (lastUpdateSensor != null && lastUpdateSensor.enabled) {
```
Comment (JBassett) [inline app/src/main/java/io/homeassistant/companion/android/sensors/SensorWorker.kt:76]:
```suggestion
            if (lastUpdateSensor?.enabled == true) {
```

File: app/src/main/java/io/homeassistant/companion/android/sensors/StepsSensorManager.kt [hunk 4]
```diff
@@ -93,16 +93,14 @@ class StepsSensorManager : SensorManager, SensorEventListener {
...
-            if (event.sensor.type == Sensor.TYPE_STEP_COUNTER) {
-                onSensorUpdated(
-                    latestContext,
-                    stepsSensor,
-                    event.values[0].roundToInt().toString(),
-                    "mdi:walk",
-                    mapOf()
-                )
-            }
+        if (event != null && event.sensor.type == Sensor.TYPE_STEP_COUNTER) {
```
Comment (JBassett) [inline app/src/main/java/io/homeassistant/companion/android/sensors/StepsSensorManager.kt:96]:
```suggestion
        if (event?.sensor.type == Sensor.TYPE_STEP_COUNTER) {

```

**Review by JBassett — APPROVED** [review]

**Comment by homeassistant** [comment]
Hi @TESTPERSONAL, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;android#1867).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by JBassett — APPROVED** [review]

**Comment by JBassett** [comment]
@nick2525 we really need to figure out your commit email issues.

**Pull request merged**